### ISSUE TITLE
Mention Ruby 2.4 requirement

### ISSUE THREAD

**Issue by gdotdesign** [soutaro/steep#2]
I tried to use steep on 2.3 and got this error:

```
#<NoMethodError: undefined method `transform_values' for {}:Hash>
  /home/gdot/.rvm/gems/ruby-2.3.4@rublem/gems/steep-0.1.0.pre/lib/steep/interface.rb:149:in `substitute'
  /home/gdot/.rvm/gems/ruby-2.3.4@rublem/gems/steep-0.1.0.pre/lib/steep/interface.rb:194:in `substitute'
  /home/gdot/.rvm/gems/ruby-2.3.4@rublem/gems/steep-0.1.0.pre/lib/steep/signature/class.rb:91:in `block (2 levels) in instance_methods'
  /home/gdot/.rvm/gems/ruby-2.3.4@rublem/gems/steep-0.1.0.pre/lib/steep/signature/class.rb:91:in `map'
  /home/gdot/.rvm/gems/ruby-2.3.4@rublem/gems/steep-0.1.0.pre/lib/steep/signature/class.rb:91:in `block in instance_methods'
  /home/gdot/.rvm/gems/ruby-2.3.4@rublem/gems/steep-0.1.0.pre/lib/steep/signature/class.rb:88:in `each'
  /home/gdot/.rvm/gems/ruby-2.3.4@rublem/gems/steep-0.1.0.pre/lib/steep/signature/class.rb:88:in `instance_methods'
  /home/gdot/.rvm/gems/ruby-2.3.4@rublem/gems/steep-0.1.0.pre/lib/steep/type_assignability.rb:227:in `resolve_interface'
  /home/gdot/.rvm/gems/ruby-2.3.4@rublem/gems/steep-0.1.0.pre/lib/steep/signature/class.rb:344:in `validate'
  /home/gdot/.rvm/gems/ruby-2.3.4@rublem/gems/steep-0.1.0.pre/lib/steep/type_assignability.rb:302:in `block in validate'
  /home/gdot/.rvm/gems/ruby-2.3.4@rublem/gems/steep-0.1.0.pre/lib/steep/type_assignability.rb:301:in `each'
  /home/gdot/.rvm/gems/ruby-2.3.4@rublem/gems/steep-0.1.0.pre/lib/steep/type_assignability.rb:301:in `validate'
  /home/gdot/.rvm/gems/ruby-2.3.4@rublem/gems/steep-0.1.0.pre/lib/steep/type_assignability.rb:14:in `initialize'
  /home/gdot/.rvm/gems/ruby-2.3.4@rublem/gems/steep-0.1.0.pre/lib/steep/drivers/check.rb:31:in `new'
  /home/gdot/.rvm/gems/ruby-2.3.4@rublem/gems/steep-0.1.0.pre/lib/steep/drivers/check.rb:31:in `run'
  /home/gdot/.rvm/gems/ruby-2.3.4@rublem/gems/steep-0.1.0.pre/lib/steep/cli.rb:76:in `process_check'
  /home/gdot/.rvm/gems/ruby-2.3.4@rublem/gems/steep-0.1.0.pre/lib/steep/cli.rb:41:in `run'
  /home/gdot/.rvm/gems/ruby-2.3.4@rublem/gems/steep-0.1.0.pre/exe/steep:11:in `<top (required)>'
  /home/gdot/.rvm/gems/ruby-2.3.4@rublem/bin/steep:22:in `load'
  /home/gdot/.rvm/gems/ruby-2.3.4@rublem/bin/steep:22:in `<main>'
  /home/gdot/.rvm/gems/ruby-2.3.4@rublem/bin/ruby_executable_hooks:15:in `eval'
  /home/gdot/.rvm/gems/ruby-2.3.4@rublem/bin/ruby_executable_hooks:15:in `<main>'
```

`transform_values` is added in 2.4 http://blog.bigbinary.com/2017/06/14/ruby-2-4-added-hash-transform-values-and-its-destructive-version-from-active-support.html

**Comment by soutaro** [comment]
Thanks, I will update README soon.

**Comment by soutaro** [comment]
Fixed in 5fd1b93b039cdf67f411696f6d8472e7c1e60a21 🎉
